loopd/utils.go

```diff
@@ -10,19 +10,26 @@ import (
 )
 
 // getClient returns an instance of the swap client.
-func getClient(config *config, lnd *lndclient.LndServices) (*loop.Client,
+func getClient(cmdCfg *config, lnd *lndclient.LndServices) (*loop.Client,
 	func(), error) {
 
-	storeDir, err := getStoreDir(config.Network)
+	storeDir, err := getStoreDir(cmdCfg.Network)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	swapClient, cleanUp, err := loop.NewClient(
-		storeDir, config.SwapServer, config.Proxy, config.Insecure,
-		config.TLSPathSwapSrv, lnd, btcutil.Amount(config.MaxLSATCost),
-		btcutil.Amount(config.MaxLSATFee),
-	)
+	cfg := &loop.ClientConfig{
```

Comment:
nit: maybe better to call this `clientConfig` and leave the other `config` or `cfg` as that fits better with the type name.

Reply:
fixed